manifests/4.6/crd-drop-status-validation-patch.yaml

```diff
@@ -0,0 +1,4 @@
+- op: replace
+  path: /spec/validation/openAPIV3Schema/properties/status
+  value:
+    type: object
```

Comment:
what was it being auto generated as before?

Reply:
We didn't have status validation at all. Thus replacing the generated one with a placeholder object.